src/styles/blocks/card.scss

```diff
@@ -0,0 +1,121 @@
+.card {
+  box-sizing: border-box;
+
+  width: 200px;
+  height: 408px;
+
+  display: flex;
+  flex-direction: column;
+
+  border-radius: 5px;
+  background-color: $primary-bg-color;
+  border: 1px solid $primary-bg-color;
+
+  &__image {
+    padding: 32px 20px 0;
+    height: 134px;
+    width: 160px;
+
+    object-fit: contain;
+    object-position: center;
+    align-self: center;
+  }
+
+  &__product {
+
+    &-name {
+      padding: 40px 16px 4px;
+      height: 36px;
+
+      line-height: 18px;
+      font-weight: 500px;
+      font-size: 12px;
+      color: $main-text-color;
+
+      transition: color $transition-duration, transform $transition-duration;
+    }
+
+    &-code {
+      height: 14px;
+      margin: 0 16px;
+      display: flex;
+      justify-content: start;
+      color: $secondary-text-color;
+      font-size: 10px;
+      line-height: 14px;
+    }
+  }
+
+  &__review {
+    height: 16px;
+    margin: 16px 16px 0;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+
+    &-count {
+      font-size: 10px;
+      line-height: 14px;
+    }
+
+    &-text {
+      color: $main-text-color;
+      font-weight: 400;
+    }
+  }
+
+  &__price {
+    height: 18px;
+    margin: 24px 16px 0;
+    display: flex;
+    justify-content: space-between;
+
+    &-text {
+      color:$secondary-text-color;
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+    }
+
+    &-tag {
+      color: $main-text-color;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      align-self:flex-start;
+    }
+  }
+
+  &__button {
+    height: 40px;
+    width: 166px;
+    margin: 16px;
+
+    font-size: 14px;
+    line-height: 16px;
+    color: $primary-bg-color;
+    text-transform: uppercase;
+    font-weight: 700;
+    text-align: center;
+
+    border: none;
+    border-radius: 5px;
+    background-color: $button-main-color;
+
+    transition-duration: $transition-duration;
```

Comment:
https://github.com/mate-academy/layout_catalog/pull/4067#discussion_r1478296207 - fix this issue in all styles.